contracts/gasProxy.sol

```diff
@@ -21,43 +21,38 @@ import "./internals/controllable.sol";
 import "./internals/gasRefundable.sol";
 
 
-contract GasProxy is Ownable, Controllable, GasRefundable {
-
-	/// @dev Controllers are used by the application.	    
+contract GasProxy is Controllable, GasRefundable {
+	/// @notice Emits the transaction executed by the controller.
 	event ExecutedTransaction(address _destination, uint256 _value, bytes _data, bytes _returnData);
```

Comment:
Do we want to have the same name that we use in the wallet for this event?

Reply:
I think so - IMO it's the same operation so it should have the same name